pr_check.sh

```diff
@@ -6,7 +6,7 @@ source "${PWD}/cicd/jenkins_env.sh"
 
 SKIP=""
 
-if [ "${HAS_PYTHON_38}" != "True" ]; then
+if ! [[ (${PYTHON_MAJOR_VERSION} -eq 3 && ${PYTHON_MINOR_VERSION} -ge 8) ]]; then
```

Comment:
what's up with pymarkdown?

Reply:
Installing with Python <3.8 results in:
```
 An unexpected error has occurred: CalledProcessError: command: ('/srv/volume/jenkins/.cache/pre-commit/repod580we6p/py_env-python3/bin/python', '-mpip', 'install', '.')
08:40:08 return code: 1
08:40:08 expected return code: 0
08:40:08 stdout:
08:40:08     Processing /srv/volume/jenkins/.cache/pre-commit/repod580we6p
08:40:08       Preparing metadata (setup.py): started
08:40:08       Preparing metadata (setup.py): finished with status 'done'
08:40:08     
08:40:08 stderr:
08:40:08     ERROR: Could not find a version that satisfies the requirement application-properties==0.5.0 (from pymarkdownlnt) (from versions: none)
08:40:08     ERROR: No matching distribution found for application-properties==0.5.0
08:40:08 
```
[application-properties](https://pypi.org/project/application-properties/) (by the same author) requires Python 3.8.

[mdformat](https://github.com/executablebooks/mdformat) is an alternative which supports GH-flavored markdown, but it requires Python >=3.7.

I don't have any issues with it locally, but if it ends up being too complex to maintain I don't mind dropping it either.